Src/InGreed.DataAccess/FakeDA/FakeIngredientDA.cs

```diff
@@ -0,0 +1,43 @@
+﻿using InGreed.DataAccess.Enums;
+using InGreed.DataAccess.Interfaces;
+using InGreed.Domain.Models;
+
+namespace InGreed.DataAccess.FakeDA;
+
+public class FakeIngredientDA : IIngredientDA
+{
+    private List<Ingredient> _ingredients = new()
+    {
+        new() { Id = 1, IconURL = string.Empty, Name = "Ender Pearl" },
+        new() { Id = 2, IconURL = string.Empty, Name = "Nether Wart" },
+        new() { Id = 3, IconURL = string.Empty, Name = "Bone Meal" }
+    };
+    private int currentId = 3;
+
+    public IngredientDAAddResponse AddToProduct(int ingredient, int productId)
+    {
+        return IngredientDAAddResponse.Success;
+    }
+
+    public int Create(Ingredient ingredient)
+    {
+        ingredient.Id = ++currentId;
```

Comment:
```suggestion
        if(GetById(ingredient.Id) return;
        ingredient.Id = ++currentId;
```
check if is already in db?